### ISSUE TITLE
Powered Lights not working

### ISSUE THREAD

**Issue by Gunner76th** [SleepyTrousers/EnderIO#272]
I tried to use the powered lights, and seem to have found a bug. If connected to the Energy Conduit 8784 , the light does not receive any power.  I have verified this, by connecting it to both a conduit coming off of the bank, as well as 1 block from my generators, and in both cases, the light failed to light up. If the light is connected to the bank or a generator directly (and a redstone signal suchas from a lever is applied) then the light ignites. It also does not seem to accept power from BC power pipes either. 

TL:DR - Powered lamps do not accept power from any conduit or pipe. Powered Lights only work if connected directly to Capacitor Bank or Generator.

Ender IO version in use - 0.4.0
